packages/babel/test/fixtures/transformation/optimisation.react.constant-elements/inline-elements/expected.js

```diff
@@ -23,4 +27,4 @@ function render() {
   return function () {
     return _ref2;
   };
-}
+}
```

Comment:
Nah it's all good. I should add a newline to the fs comment though to avoid it in the future.